### PR TITLE
Update telemetry privacy document for UI interaction and exceptions

### PR DESCRIPTION
**Pull request by robertbrignull**
Updates the document describing what telemetry we collect from the CodeQL VS Code extension.

Don't be afraid to suggest rewriting what I've put here. I wasn't really sure how to fit in with the existing style. I also wasn't sure how to split information between the "what we include" and "what we don't include" sections.

@jf205 and @github/code-scanning-secexp

### CODE REVIEW

File: docs/codeql/codeql-for-visual-studio-code/about-telemetry-in-codeql-for-visual-studio-code.rst [hunk 1]
```diff
@@ -20,7 +20,12 @@ What data is collected
 If you opt in, GitHub collects the following information related to the usage of the extension. The data collected are:
 
 - The identifiers of any CodeQL-related VS Code commands that are run.
-- For each command: the timestamp, time taken, and whether or not the command completed successfully.
+    - For each command: the timestamp, time taken, and whether or not the command completed successfully.
+- Usage of UI elements, including buttons, links, and other inputs.
+    - The contents of links or text inputs is not recorded.
```
Comment (aeisenberg) [inline docs/codeql/codeql-for-visual-studio-code/about-telemetry-in-codeql-for-visual-studio-code.rst:25]:
```suggestion
    - Link targets and text inputs are not recorded.
```

File: docs/codeql/codeql-for-visual-studio-code/about-telemetry-in-codeql-for-visual-studio-code.rst [hunk 2]
```diff
@@ -20,7 +20,12 @@ What data is collected
 If you opt in, GitHub collects the following information related to the usage of the extension. The data collected are:
 
 - The identifiers of any CodeQL-related VS Code commands that are run.
-- For each command: the timestamp, time taken, and whether or not the command completed successfully.
+    - For each command: the timestamp, time taken, and whether or not the command completed successfully.
+- Usage of UI elements, including buttons, links, and other inputs.
```
Comment (aeisenberg) [inline docs/codeql/codeql-for-visual-studio-code/about-telemetry-in-codeql-for-visual-studio-code.rst:24]:
```suggestion
- Interactions with UI elements, including buttons, links, and other inputs.
```

File: docs/codeql/codeql-for-visual-studio-code/about-telemetry-in-codeql-for-visual-studio-code.rst [hunk 3]
```diff
@@ -20,7 +20,12 @@ What data is collected
...
+    - The contents of links or text inputs is not recorded.
+    - Mouse movement and hovering is not recorded.
```
Comment (aeisenberg) [inline docs/codeql/codeql-for-visual-studio-code/about-telemetry-in-codeql-for-visual-studio-code.rst:26]:
```suggestion
    - Mouse movement and hovering are not recorded.
```

File: docs/codeql/codeql-for-visual-studio-code/about-telemetry-in-codeql-for-visual-studio-code.rst [hunk 4]
```diff
@@ -20,7 +20,12 @@ What data is collected
...
+- Occurrence of exceptions and error.
```
Comment (aeisenberg) [inline docs/codeql/codeql-for-visual-studio-code/about-telemetry-in-codeql-for-visual-studio-code.rst:27]:
```suggestion
- Occurrence of exceptions and errors.
```

File: docs/codeql/codeql-for-visual-studio-code/about-telemetry-in-codeql-for-visual-studio-code.rst [hunk 5]
```diff
@@ -20,7 +20,12 @@ What data is collected
...
+    - All sensitive information such as file paths or non-static exception message content is removed before uploading.
```
Comment (aeisenberg) [inline docs/codeql/codeql-for-visual-studio-code/about-telemetry-in-codeql-for-visual-studio-code.rst:28]:
```suggestion
    - All sensitive information such as file paths and non-static exception message content is removed before uploading.
```

File: docs/codeql/codeql-for-visual-studio-code/about-telemetry-in-codeql-for-visual-studio-code.rst [hunk 6]
```diff
@@ -45,6 +50,7 @@ We only collect the minimal amount of data we need to answer the questions about
 - No CodeQL database names or contents
 - No contents of CodeQL queries
 - No filesystem paths
+- No user-input text
```
Comment (aeisenberg) [inline docs/codeql/codeql-for-visual-studio-code/about-telemetry-in-codeql-for-visual-studio-code.rst:53]:
```suggestion
- No user-input text
- Mouse interactions, such as movement and hovers
```

**Review by aeisenberg — COMMENTED** [review]
The content looks mostly right to me. I have a few copy-edit suggestions. A pass through by James would still be a good idea.

**Comment by robertbrignull** [comment]
Thanks @aeisenberg. I've applied those suggestions. I agree a pass through from @jf205 is still needed ideally.

**Review by aeisenberg — APPROVED** [review]
LGTM, but good to get a second opinion.

**Review by jf205 — APPROVED** [review]
Thanks @robertbrignull, LGTM 👍🏻

**Pull request merged**